setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import find_packages, setup
 
 
-install_requires = ["Django>=1.11,<4.1"]
+install_requires = ["Django>=1.11"]
```

Comment:
It's really inadvisable to set upper version bounds:  https://iscinumpy.dev/post/bound-version-constraints/ . TL;DR you block attempts to upgrade for little extra safety.